lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
+
+        parser.on('--force-color', '--force-colour', 'Force the output to be in color.') do |_o|
+          options[:force_color] = true
+          options.delete(:no_color)
```

Comment:
OK, this is in. I actually check at the `configuration.rb` options level instead of at the flag-parsing level, so that the issue will be caught whether the user is using command-line flags or setting the configs directly. However, I'm running into an issue where `abort` is being called twice:

```
~/apps/battle_logic [master] # bin/rspec --force-color --no-color
Please use only one of --force-color and --no-color

Please use only one of --force-color and --no-color
```

I hate to ask you to do my debugging for me, but could you check out where that's implemented and see if a cause or a better place to put that `abort` jumps out at you? Here it is:

https://github.com/rspec/rspec-core/pull/2321/commits/e8a31a6ebfdcc82ee1af13f5b0e8a345c6f8aa25